src/pybind/mgr/rgw/module.py

```diff
@@ -42,7 +43,17 @@ def list_daemons(self, service_name, daemon_type = None, daemon_id = None, host
 
 
 class Module(orchestrator.OrchestratorClientMixin, MgrModule):
-    MODULE_OPTIONS = []
+    MODULE_OPTIONS = [
+        Option(name='usage_trim_realms',
+               default='',
+               desc='Comma separated list with realms to usage trim'),
+        Option(name='usage_trim_older_than_days',
+               default=0,
+               desc='Trim usage log entries older than this amount of days, must be set to a positive value'),
+        Option(name='usage_trim_interval',
+               default=43200,
+               desc='Interval between trimming usage log entries, defaults to 12 hours'),
```

Comment:
We need to specify the unit of this configuration parameter. Do we really need seconds? isn't hours enough in this case?

Reply:
I will fix this!